### ISSUE TITLE
Can't log in to oss-fuzz.com with Safari/macOS

### ISSUE THREAD

**Issue by nwellnhof** [google/oss-fuzz#8481]
When trying to log in to oss-fuzz.com with my Google account using Safari on macOS, I'm only seeing a box with a loading animation and nothing happens.

Safari Version 16.0 (17614.1.25.9.10, 17614)
macOS Monterey 12.6

**Comment by oliverchang** [comment]
Hmm, this works for me on Chrome across various platforms. Do you see anything in the console log output? 

@Alan32Liu @Navidem do any of you have a macOS device to test this?

**Comment by nwellnhof** [comment]
Here's what I see on the console:

[Info] Successfully preconnected to https://fonts.gstatic.com/
[Error] Failed to load resource: the server responded with a status of 404 () (webcomponents-lite.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (apply-shim.min.js.map, line 0)
[Error] Failed to load resource: the server responded with a status of 404 () (custom-style-interface.min.js.map, line 0)
[Error] Refused to load data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIj8+Cjxzdmcgd2lkdGg9IjEyIiBoZWlnaHQ9IjQiIHZpZXdQb3J0PSIwIDAgMTIgNCIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPgogIDxlbGxpcHNlIGN4PSIyIiBjeT0iMiIgcng9IjIiIHJ5PSIyIj4KICAgIDxhbmltYXRlIGF0dHJpYnV0ZU5hbWU9ImN4IiBmcm9tPSIyIiB0bz0iLTEwIiBkdXI9IjAuNnMiIHJlcGVhdENvdW50PSJpbmRlZmluaXRlIiAvPgogIDwvZWxsaXBzZT4KICA8ZWxsaXBzZSBjeD0iMTQiIGN5PSIyIiByeD0iMiIgcnk9IjIiIGNsYXNzPSJsb2FkZXIiPgogICAgPGFuaW1hdGUgYXR0cmlidXRlTmFtZT0iY3giIGZyb209IjE0IiB0bz0iMiIgZHVyPSIwLjZzIiByZXBlYXRDb3VudD0iaW5kZWZpbml0ZSIgLz4KICA8L2VsbGlwc2U+Cjwvc3ZnPgo= because it does not appear in the img-src directive of the Content Security Policy.
[Error] Failed to load resource: the server responded with a status of 403 () (report-csp-failure, line 0)
[Error] Unhandled Promise Rejection: TypeError: Right side of assignment cannot be destructured
	(anonymous function)
	rejectPromise
	promiseReactionJob

**Comment by Alan32Liu** [comment]
Yep, it works on Chrome but not on Safari because of the same error.
I've tried logging in with my GitHub account and Google account, but both of them failed for the same reason.
Also tried logging in in the private/incognito mode, which did not work either.

**Comment by Alan32Liu** [comment]
I guess this relates to login failures?
Clicking on the side menu items (e.g., Testcases, Fuzzer Statistics, etc.) brought me back to the login page (to choose whether to log in with Google or GitHub account).

**Comment by marka63** [comment]
Same behaviour with Safari Version 16.0 (16614.1.25.9.10, 16614) on BigSur 11.7
